Doc/faq/library.rst

```diff
@@ -670,7 +670,7 @@ A summary of available frameworks is maintained by Paul Boddie at
 https://wiki.python.org/moin/WebProgramming\ .
 
 Cameron Laird maintains a useful set of pages about Python web technologies at
-http://phaseit.net/claird/comp.lang.python/web_python.
+http://web.archive.org/web/20210224183619/http://phaseit.net/claird/comp.lang.python/web_python.
```

Comment:
Should we just remove this link? The page seems to be very outdated.

Reply:
> For the web.archive links I think http could be replaced with https, not sure how important it is though.

Thank you, applied. Google prioritized the http version so initially I took whatever it gave.

> Should we just remove this link? The page seems to be very outdated.

I believe it goes out of this PR's scope so it deserves a separated PR.